### PR TITLE
Implement test suite for Carousel

### PR DESCRIPTION
**Pull request by chujacky**

### CODE REVIEW

File: client/src/components/Suggestions.jsx [hunk 1]
```diff
@@ -18,13 +18,12 @@ class Suggestions extends React.Component {
...
       .then((response) => {
-        // const data = [];
-        // while (response.data.length) {
-        //   data.push(response.data.splice(0, 3));
-        // }
-        console.log(response.data);
+        const data = [];
+        for (let i = 0; i < response.data.length; i += 1) {
+          data.push(response.data[i][0]);
+        }
```
Comment (DymaxionMap) [inline client/src/components/Suggestions.jsx]: If you're just iterating through all the elements of `response.data`, then using `map` might be cleaner.

File: client/src/components/suggestions.test.js [hunk 2]
```diff
@@ -6,7 +6,7 @@ import Carousel from './Carousel.jsx';
 
 Enzyme.configure({ adapter: new Adapter() });
 
-describe('Suggesion Component', () => {
+describe('Suggesions Component', () => {
```
Comment (DymaxionMap) [inline client/src/components/suggestions.test.js]: You left out the "t" in "Suggestions"

File: client/src/components/Suggestions.jsx (hunk 1 above)
Comment (chujacky) [reply client/src/components/Suggestions.jsx]: great catch! I will implement that

**Comment by DymaxionMap** [comment]
Looks good

**Pull request merged**